src/pages/Profile.jsx

```diff
@@ -32,29 +35,45 @@ function Profile() {
     },
   ];
 
+  const [user, setUser] = useState({});
+
+  useEffect(async () => {
+    const res = await AFCBackend.get(`/users/${userId}`, {
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    });
+    if (res.status === 200) {
+      setUser(res.data);
+    }
+  }, [userId]);
+
   return (
     <div>
-      <p>This is the volunteer profile page - USER ID OF VOLUNTEER HARDCODED TEMPORARILY</p>
-      <h1>[VOLUNTEER NAME HERE]s Profile</h1>
+      <h1 id="profile-heading">
```

Comment:
let's prefer using classes to ids everywhere, generally specific classnames > ids